frontend/packages/dev-console/integration-tests/tests/pipeline.scenario.ts

```diff
@@ -120,4 +137,59 @@ describe('Pipeline', async () => {
       true,
     );
   });
+
+  it('run pipeline and generate pipeline logs scenario', async () => {
+    let runName: string;
+    await switchPerspective(Perspective.Developer);
+    await browser.wait(until.visibilityOf(pageSidebar));
+    await pipelinecheckStatus();
+    await execSync(
+      `oc create -f ./packages/dev-console/integration-tests/views/sample-app-pipeline.yaml -n ${testName}`,
+    );
+    await browser.wait(until.visibilityOf(pipelineTableBody));
+    expect(pipelineTableBody.samplePipelineElement.isDisplayed()).toBe(true);
```

Comment:
This is not a good idea. `pipelineTableBody` references the internal working of the table:
```
export const pipelineTableBody = element(
  by.className('ReactVirtualized__VirtualGrid__innerScrollContainer'),
);
```

This is extremely volatile.

Reply:
But it's referring to the internal part of the table so I think pipelineTableBody is appropriate here.